ristonrodrigues723.md

```diff
@@ -0,0 +1,172 @@
+| 4×4 Matrix Keypad Membrane Switch for Arduino, ARM and other MCU | [https://robu.in/product/4x4-matrix-keypad-membrane-switch-arduino-arm-mcu/](https://robu.in/product/4x4-matrix-keypad-membrane-switch-arduino-arm-mcu/) | $1.09|
+| 	Standard 5V 3A Power Supply with 5.5mm DC Plug|[(https://robu.in/product/orange-5v-3a-power-supply-adapter-charger-with-5-5mm-dc-plug/](https://robu.in/product/orange-5v-3a-power-supply-adapter-charger-with-5-5mm-dc-plug/)| $2.78|
+| GL-12 840 Points Solderless Breadboard X 2 | [https://robu.in/product/breadboard-840-tie-points-solderless-diy-project-circuit-test-breadboard/](https://robu.in/product/breadboard-840-tie-points-solderless-diy-project-circuit-test-breadboard/) | $1.45|
+| 	Female to Female DuPont Line 40 Pin 30cm | [https://robu.in/product/female-female-dupont-line-40-pin-30cm/](https://robu.in/product/female-female-dupont-line-40-pin-30cm/)| $0.60|
+| HC-SR04-Ultrasonic Range Finder | [https://robu.in/product/hc-sr04-ultrasonic-range-finder/](https://robu.in/product/hc-sr04-ultrasonic-range-finder/) | $0.87|
+| Electronics ESP32 ESP-32 ESP-32S ESP 32 Development Board CP2102 WiFi Bluetooth Ultra-Low Power Consumption Dual Core X 5 | [https://www.amazon.in/gp/product/B07TYCFX5C/ref=ewc_pr_img_3?smid=A2KWJ4IHWZH3FB&psc=1](https://www.amazon.in/gp/product/B07TYCFX5C/ref=ewc_pr_img_3?smid=A2KWJ4IHWZH3FB&psc=1) | $34.75|
+| Raindrops Detection Sensor Module	 |[https://robu.in/product/raindrops-detection-sensor-module-rain-weather-humidity/](https://robu.in/product/raindrops-detection-sensor-module-rain-weather-humidity/)| $0.91|
+| 40 Pin Red GPIO Extension Board for Raspberry Pi|[https://robu.in/product/40-pin-red-gpio-extension-board-for-raspberry-pi/](https://robu.in/product/40-pin-red-gpio-extension-board-for-raspberry-pi/) | $1.50|
+| 20 CM 40 Pin Dupont Cable Male/Male, Male/Female, Female/Female Cable Combo | [https://robu.in/product/20-cm-40-pin-dupont-male-male-male-female-female-female-cable-combo/](https://robu.in/product/20-cm-40-pin-dupont-male-male-male-female-female-female-cable-combo/) | $1.50|
+|Soil Moisture Meter, Soil Humidity Sensor, Water Sensor, Soil Hygrometer for Ardunio X 2 | https://robu.in/product/soil-moisture-meter-soil-humidity-sensor-water-sensor-soil-hygrometer-ardunio/ | $1.45 |
+|Flame Sensor Module |[https://robu.in/product/flame-sensor-module/](https://robu.in/product/flame-sensor-module/) | $0.85 |
+|RPI4-MODBP-8GB| https://in.element14.com/raspberry-pi/rpi4-modbp-8gb/raspberry-pi-4-model-b-cortex/dp/3369503) | $73.79 |
+| I2C ADS1115 16 Bit ADC 4 channel Module with Programmable Gain Amplifier | [https://robu.in/product/i2c-ads1115-16-bit-adc-4-channel-module-with-programmable-gain-amplifier/](https://robu.in/product/i2c-ads1115-16-bit-adc-4-channel-module-with-programmable-gain-amplifier/) | $9.07|
+|37-1K - Resistance:1kohm |[https://in.element14.com/mcm/37-1k/resistance-1kohm/dp/2784369](https://in.element14.com/mcm/37-1k/resistance-1kohm/dp/2784369) | $2.14 |
```

Comment:
do you need more resistors or do you plan on reusing components?

Reply:
oh i actually will buy some
and i have some of them already present old kits

Reply:
also reusing components i plan on doing that greatly

Reply:
yes but i would reuse parts as i mentioned earlier and i have some resistors lying around from old projects, also can borrow some from friends